### PR TITLE
Qo l mods

### PR DESCRIPTION
**Pull request by DeRampage**
Implemented a lot of mods.
started translation and structuring

### CODE REVIEW

File: addons/BC_main/$PBOPREFIX$ [hunk 1]
```diff
@@ -0,0 +1 @@
+z\bcdw\addons\main
```
Comment (Desty84) [inline addons/BC_main/$PBOPREFIX$]: Der Pfad schaut falsch aus

File: addons/BC_main/functions/script_component.hpp [hunk 2]
```diff
@@ -0,0 +1 @@
+#include "\z\bcdw\addons\main\script_component.hpp"
```
Comment (Desty84) [inline addons/BC_main/functions/script_component.hpp]: Pfad, siehe oben

File: addons/BC_wedge/$PBOPREFIX$ [hunk 3]
```diff
@@ -0,0 +1 @@
+z\bcdw\addons\wedge
```
Comment (Desty84) [inline addons/BC_wedge/$PBOPREFIX$]: Pfad

**Review by Desty84 — CHANGES_REQUESTED** [review]
Check Paths ... marked 3, but seems like most BC stuff uses wrong pathes

File: addons/BC_wedge/$PBOPREFIX$ (hunk 3 above)
Comment (DeRampage) [reply addons/BC_wedge/$PBOPREFIX$]: done

File: addons/BC_main/functions/script_component.hpp (hunk 2 above)
Comment (DeRampage) [reply addons/BC_main/functions/script_component.hpp]: also done

File: addons/BC_main/$PBOPREFIX$ (hunk 1 above)
Comment (DeRampage) [reply addons/BC_main/$PBOPREFIX$]: totally fuckin done

**Review by Desty84 — APPROVED** [review]

**Pull request merged**